### DISCUSSION TITLE
Questions to the 3rd phase and final submission

### DISCUSSION THREAD

**Discussion by bqpeng-1006** [PaddlePaddle/PaddleSpatial#254] [BaiduKDDCup2022] [answered]
Dear organizers,

I have some questions toward the final submission since the upcoming deadline. 
1.  Is the deadline July 15 AOE sharp? 
2. Which submission will count for the final score, e.g. the last one?
3. Will the 3rd phase be a private comparison blind to all the competitors, or final rankings will be released immediately once the submission is finished ?

Thank you,

**Comment by ramber1836** [comment]
The leaderboard in the 3rd phase will be open for competitors. 
The instructions about the Phase 3 competition will be released.

**Comment by zhoujingbo** [comment, marked as answer]
From July 16th 00:01 AM to July 18th 11: 59 PM (AOE time), we will go to Phase 3 of the Baidu KDD Cup. During this phase, the test data will be updated. Each team can submit its prediction model three times per day. During this phase, the organizer team will not answer any questions about submission errors (except for the Pending Timeout error) in the discussion forum. The leaderboard in Phase 3 will determine the candidate awardee teams.
